scripts/artifacts.sh

```diff
@@ -0,0 +1,48 @@
+#! /bin/sh
+
+# check parameter
+if [ $# -ne 1 ]; then
+  echo "Usage: $0 <release_tag>"
+  exit 1
+fi
+
+# set the home path
+HOME_PATH=$(
+  cd "$(dirname "$0")"
+  cd ..
+  pwd
+)
+echo "HOME_PATH: ${HOME_PATH}"
+
+# helm package
+cd $HOME_PATH/helm-charts/charts/kube-starrocks/
+# must be executed before helm index operation
+helm package .
+
+# parse the package name
+# get the name from the path, e.g. kube-starrocks-1.6.1.tgz
+package_name=$(ls -t $HOME_PATH/helm-charts/charts/kube-starrocks/*.tgz | head -1 | awk -F '/' '{print $NF}')
+# remove the suffix from package_name, e.g. kube-starrocks-1.6.1
+package_name=${package_name%.tgz}
```

Comment:
is it more reliable to parse the name and package_name info directly from `Chart.yaml`?